ocaml/tests/test_client.ml

```diff
@@ -0,0 +1,42 @@
+
+(** This module tests more layers of xapi than other tests by using the Client
+    module to make XenAPI calls. It tests many of the auto-generated files as
+    these XenAPI calls go through the client, server.ml, message forwarding,
+    and database layers. *)
+
```

Comment:
I assume the difference between the target of these tests and of quicktests is that this actually tests the mechanism the quicktests use?

Reply:
For me, the main point of this is that this is a unit test, so I can catch issues in the autogenerated code by simply running `make test` :smile: 
But yes, I think it does test that mechanism.